lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -1068,6 +521,16 @@ module Resources
               'display': 'Emergency Contact'
             }
           ]
+        },
+        {
+          'extension': [
+            {
+              'valuePeriod': {
+                'start': '2019-12-26T16:06:26.000Z'
+              },
+              'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            }
+          ]
```

Comment:
We don't support relationship period on write, so we shouldn't add this to the create body.

Reply:
removed it here https://github.com/cerner/fhir.cerner.com/pull/624/commits/906c8894c70e01bd63fcd1eae7377f2bd18849c5#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eL524-L533